sysbox-in-docker/Makefile

```diff
@@ -81,6 +94,13 @@ ubuntu-bionic: ## Build Ubuntu-Bionic docker image
 	@printf "$(LAUNCH_INSTRUCTION) "
 	@printf "$(SIND_IMAGE):$@\n\n"
 
+ubuntu-eoan: ## Build Ubuntu-Eoan docker image
```

Comment:
See my previous comment, why adding EoL'd releases?

Reply:
Same as before (my dev machine is eoan).

Reply:
Let's push this Eoan Dockerfile for now and discuss what we'll do in the future for unsupported distro-releases such as this one.